tests/subsys/fs/multi-fs/src/main.c

```diff
@@ -10,28 +10,36 @@
 #include "test_littlefs.h"
 #include "test_fs_shell.h"
 
-void test_main(void)
+ZTEST(multi_fs_fat, test_multi_fs_fat)
 {
-	ztest_test_suite(multifs_fs_test,
-			 ztest_unit_test(test_clear_flash),
-			 ztest_unit_test(test_fat_mount),
-			 ztest_unit_test(test_littlefs_mount),
-			 ztest_unit_test(test_fat_mkdir),
-			 ztest_unit_test(test_littlefs_mkdir),
-			 ztest_unit_test(test_fat_readdir),
-			 ztest_unit_test(test_littlefs_readdir),
-			 ztest_unit_test(test_fat_rmdir),
-			 ztest_unit_test(test_littlefs_rmdir),
-			 ztest_unit_test(test_fat_open),
-			 ztest_unit_test(test_littlefs_open),
-			 ztest_unit_test(test_fat_write),
-			 ztest_unit_test(test_littlefs_write),
-			 ztest_unit_test(test_fat_read),
-			 ztest_unit_test(test_littlefs_read),
-			 ztest_unit_test(test_fat_close),
-			 ztest_unit_test(test_littlefs_close),
-			 ztest_unit_test(test_fat_unlink),
-			 ztest_unit_test(test_littlefs_unlink),
-			 ztest_unit_test(test_fs_help));
-	ztest_run_test_suite(multifs_fs_test);
+	test_fat_mkdir();
+	test_fat_readdir();
+	test_fat_rmdir();
+	test_fat_open();
+	test_fat_write();
+	test_fat_read();
+	test_fat_close();
+	test_fat_unlink();
+	test_fs_help();
 }
+ZTEST(multi_fs_littlefs, test_multi_fs_littlefs)
+{
+	test_littlefs_mkdir();
```

Comment:
Can we add a comment explicitly calling these out as order-dependent? After that LGTM.

Reply:
Please address the comments here, @shuai1x .

Reply:
I have modified it. Please have a look again.

Reply:
I have modified it again. And after I modified it, these tests are crisply order-dependent, so they have to be executed in order. And do I need to add a comment to explain it?

Reply:
> (...) And do I need to add a comment to explain it?

No. We probably need another PR to redesign the test to remove the dependencies.

Reply:
So in this PR or a future PR? In between the second PR, I'd rather just have a comment indicating it so it gets done and is visible.

Reply:
Why can't we just add a comment here for now and then remove it in a followup PR? Otherwise just completely fix it in a new PR.

Reply:
I have added the comment to explain it. Please have a look again. Thanks. @aaronemassey